bundles/org.openhab.binding.solax/src/test/java/org/openhab/binding/solax/internal/TestX1HybridG4Parser.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.solax.internal;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.junit.jupiter.api.Test;
+import org.openhab.binding.solax.internal.connectivity.rawdata.LocalConnectRawDataBean;
+import org.openhab.binding.solax.internal.model.InverterData;
+import org.openhab.binding.solax.internal.model.InverterType;
+import org.openhab.binding.solax.internal.model.SinglePhaseInverterData;
+import org.openhab.binding.solax.internal.model.parsers.RawDataParser;
+
+/**
+ * The {@link TestX1HybridG4Parser} Simple test that tests for proper parsing against a real data from the inverter
+ *
+ * @author Konstantin Polihronov - Initial contribution
+ */
+@NonNullByDefault
+public class TestX1HybridG4Parser {
+
+    private static final String RAW_DATA = "{\"sn\":\"SOME_SERIAL_NUMBER\",\"ver\":\"3.008.10\",\"type\":15,\"Data\":["
+            + "2388,21,460,4998,4483,4483,10,1,487,65," // [0-9]
```

Comment:
Consider using a Text Block here for improved readability and reduced risk of escaping mistakes, something like:
```java
private static final String RAW_DATA = """
    {"sn":"SOME_SERIAL_NUMBER","ver":"3.008.10","type":15,"Data":[
    2388,21,460,4998,4483,4483,10,1,487,65,
    etc.
    """
```

Reply:
Really nice feature. Never used that before. (says a developer working still on Java8) :( 
Point taken!